dev-support/docker/Dockerfile

```diff
@@ -0,0 +1,53 @@
+#
```

Comment:
i think, it should be under `packaging/src/docker`

Reply:
I agree, this looks better in the packaging project

Reply:
Done, and added a maven command to build from source, cc @ayushtkn